dev/unused-images.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e
+
+temp_dir=$(mktemp -d)
+trap 'rm -rf "$temp_dir"' EXIT
+
+# Image files
+git ls-files docs/source | grep -E "\.(png|gif|jpg|jpeg)$" > "$temp_dir/image_files.txt"
+# Files where images are referenced
+git ls-files '*.rst' '*.py' '*.html' '*.md' > "$temp_dir/tracked_files.txt"
```

Comment:
```suggestion
git ls-files '*.rst' '*.py' '*.html' '*.md' > "$temp_dir/tracked_files.txt"
```
Is it possible that `.py` file contains image?